src/script/standard.h

```diff
@@ -28,33 +28,6 @@ class CScriptID : public uint160
 static const unsigned int MAX_OP_RETURN_RELAY = 80;      //! bytes
```

Comment:
As explained in the PR:
"This will also open the door to some cleaning up on the includes.
nMaxDatacarrierBytes and MAX_OP_RETURN_RELAY are not moved from script/standard.o to policy.o because policy.o is in server while script/standard.o is in common and wouldn't link."

It can be done later, see https://github.com/jtimon/bitcoin/commit/d66803ee06175d71ff7605ed1cfa8b91aec23f62
I mean, I can move MAX_OP_RETURN_RELAY without moving nMaxDatacarrierBytes...